subsquid/scripts/init.sh

```diff
@@ -1,7 +1,44 @@
 #!/bin/sh
 
+set -eou pipefail
+
+# main entry point for the script
+main() {
+  #  ${foo:="default value if unset"}
+  if ${TRIGGER_REPROCESSING:=false} ; then
```

Comment:
Shouldn't this be `true`?

Reply:
please see the comment from above:
`  #  ${foo:="default value if unset"}`   this means set a default value if this env var is unset
if `TRIGGER_REPROCESSING` is set, then the set value if used

Reply:
Perfect!